deploy/common/k8sbased.sh

```diff
@@ -620,10 +624,19 @@ function launch_sensor {
 
     if [[ -n "$POD_SECURITY_POLICIES" ]]; then
         extra_config+=("--enable-pod-security-policies=${POD_SECURITY_POLICIES}")
+        scanner_extra_config+=("--enable-pod-security-policies=${POD_SECURITY_POLICIES}")
+    fi
+
+    if [[ "${ORCH}" == "openshift" ]]; then
+      if [[ "${ROX_OPENSHIFT_VERSION:-}" == "4" ]]; then
+        extra_config+=("--openshift-version=${ROX_OPENSHIFT_VERSION}")
+        scanner_extra_config+=("--cluster-type=openshift4")
+      fi
     fi
 
-    # Delete path
-    rm -rf "$k8s_dir/sensor-deploy"
+    # Delete paths
```

Comment:
Maybe just drop the useless comment instead?

Reply:
ok